harness/determined/common/experimental/determined.py

```diff
@@ -582,3 +583,48 @@ def stream_trials_validation_metrics(
             stacklevel=2,
         )
         return trial._stream_validation_metrics(self._session, trial_ids)
+
+    def describe_token(self, token_id: int) -> token.AccessToken:
+        """
+        Get the :class:`~determined.experimental.Token` representing the
+        token info with the provided token ID.
+        """
+        resp = bindings.get_GetAccessTokens(
+            session=self._session, tokenIds=[token_id], showInactive=True
+        )
+        return token.AccessToken._from_bindings(resp.tokenInfo, self._session)
+
+    def describe_tokens(self, token_ids: List[int]) -> token.AccessToken:
+        """
+        Get the :class:`~determined.experimental.Token` representing list of
+        token info with the provided token IDs.
+        """
+        resp = bindings.get_GetAccessTokens(
+            session=self._session, tokenIds=token_ids, showInactive=False
+        )
+        return token.AccessToken._from_bindings(resp.tokenInfo, self._session)
+
+    def list_tokens(
```

Comment:
i think all 3 of these new methods: `describe_token`, `describe_tokens`, and `list_tokens` should be combined into a singular `list_tokens` that takes in all the parameters. 

the CLI and SDK roughly overlap in functionality, but the methods shouldn't be an exact replica. in the CLI, we have to be careful about which methods to expose and how because the UI is limited to the command line, so we generally value convenience and modularity. but the SDK has a bit of a different philosophy. it's in code and made for developers, so the methods we expose can be more powerful and more robust. (see `list_experiments` here as an example)

the functionality in `describe_token`, `describe_tokens`, and `list_tokens` can be captured with a single `list_tokens` method, and the user can pass in exactly what they want, just like in code. i'm thinking `list_tokens` would have a signature like:

```python3
def list_tokens(
        self,
        username: Optional[str],
        token_ids: Optional[List[int]],
        include_inactive: bool,
        sort_by: token.TokenSortBy = token.TokenSortBy.NAME,
        order_by: OrderBy = OrderBy.ASCENDING,
    ) -> List[token.Token]:
```

this is nice because it basically mirrors the actual bindings API call, the user isn't limited by separate methods, and the method is representative of the actual query we make in the system. we should also include sort/order as accepted parameters.